recipes/openeo/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "openeo" %}
+{% set version = "0.14.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Open-EO/openeo-python-client/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 63b244ee6537e5beb262be7fb696c76af5f87a57126f2ee51277818aae0bb498
+#  git_url: https://github.com/Open-EO/openeo-python-client.git
+
+build:
+  number: 0
+#  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Best to just remove commented stuff. Also, since this is pure Python, it can be noarch:
```suggestion
  number: 0
  noarch: python
  script: "{{ PYTHON }} -m pip install . -vv"
```